src/main/java/com/locust/operator/controller/config/SysConfig.java

```diff
@@ -46,4 +58,31 @@ public class SysConfig {
     @Property(name = "config.load-generation-pods.taintTolerations.enableCrInjection")
     private boolean tolerationsCrInjectionEnabled;
 
+    /**
+     * Value configured for setting Kubernetes Jobs' ttlSecondsAfterFinished property.
+     * This method will try to convert the value to an integer or fail and report invalid values.
+     * {@code null} or empty strings will result in a {@code null} return.
+     *
+     * @return either {@code null} or an integer value greater than or equal to 0
+     */
+    public Integer getTtlSecondsAfterFinished() {
+        final Object value = this.ttlSecondsAfterFinished;
+        final String stringValue = String.valueOf(value);
+        final Integer ttl;
+
+        if (NumberUtils.isDigits(stringValue)) {
+            ttl = Integer.parseInt(stringValue);
+        } else if (StringUtils.isNotEmpty(stringValue)) {
+            throw new IllegalArgumentException(
+                String.format(
+                    "Invalid value '%s' for property ttl-seconds-after-finished",
+                    value
+                )
+            );
+        } else {
+            ttl = null;
+        }
```

Comment:
This maybe a preference based point (with slight optimisation objectives), thus it is not critical but it would be nice to have. 

Regarding the validation logic flow, wouldn't it be more efficient and feel more "natural" if the possible states of `ttl` are handled first and then the exception is thrown if needed? 
This wold mean; checking for default state and return null (which should be the one hit the most), checking if numeric value is provided and return that, else, throw exception. 

a proposal can be: 
```java 
if (NumberUtils.isDigits(stringValue)) {
        return Integer.parseInt(stringValue);
    } else if (stringValue.isEmpty()) {
        return null;
    } else {
        throw new IllegalArgumentException(
            String.format(
                "Invalid value '%s' for property ttl-seconds-after-finished",
                value
            )
        );
    }
```